ecosystem/platform/server/Gemfile

```diff
@@ -110,6 +110,10 @@ gem 'statesman', '~> 10.0'
 gem 'flipper', '~> 0.24.1'
 gem 'flipper-active_record', '~> 0.24.1'
 
+# Redis
+gem 'hiredis', '~> 0.6.3'
```

Comment:
this is a very old version; why?

Reply:
It's the current version - https://rubygems.org/gems/hiredis